lib/mpi/service.rb

```diff
@@ -166,13 +168,21 @@ def create_add_message(user_identity)
     end
 
     def create_profile_message(user_identity, search_type: MPI::Constants::CORRELATION_WITH_RELATIONSHIP_DATA)
-      return message_icn(user_identity, search_type) if user_identity.mhv_icn.present?
-      return message_edipi(user_identity, search_type) if user_identity.edipi.present? && Settings.mvi.edipi_search
+      return message_icn(user_identity, search_type) if mhv_icn_present?(user_identity)
```

Comment:
I don't think we need the changes in this file anymore, do we?

Reply:
We do because of the health care application. Specifically [these lines](https://github.com/department-of-veterans-affairs/vets-api/blob/d8f083a5a1c66f480f2c57573d0f3d845579ff10/app/models/health_care_application.rb#L118-L139) which are using the `UserAttributes` class to format data and call `MPI`.

We could take a similar approach and refactor it to use `UserIdentity` like the other instances, but it's a little more involved in this case.